docs/src/mdx/mermaid.tsx

```
@@ -0,0 +1,43 @@
+'use client';
+
+import type { MermaidConfig } from 'mermaid';
+import { useTheme } from 'next-themes';
+import { useEffect, useId, useRef, useState } from 'react';
+
+export function Mermaid({ chart }: { chart: string }) {
+	const id = useId();
+	const [svg, setSvg] = useState('');
+	const containerRef = useRef<HTMLDivElement>(null!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)